But feature `graphics` build failed with the following error:
```
CMake Error at F:/PRreview/vcpkg/downloads/tools/cmake-3.29.0-windows/cmake-3.29.0-windows-i386/share/cmake-3.29/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find PNG (missing: PNG_LIBRARY PNG_PNG_INCLUDE_DIR)
Call Stack (most recent call first):
  F:/PRreview/vcpkg/downloads/tools/cmake-3.29.0-windows/cmake-3.29.0-windows-i386/share/cmake-3.29/Modules/FindPackageHandleStandardArgs.cmake:600 (_FPHSA_FAILURE_MESSAGE)
  F:/PRreview/vcpkg/downloads/tools/cmake-3.29.0-windows/cmake-3.29.0-windows-i386/share/cmake-3.29/Modules/FindPNG.cmake:177 (find_package_handle_standard_args)
  F:/PRreview/vcpkg/scripts/buildsystems/vcpkg.cmake:859 (_find_package)
  graphics/meshes/CMakeLists.txt:1 (find_package)
```

--- a/ports/awlib/vcpkg.json
+++ b/ports/awlib/vcpkg.json
@@ -4,7 +4,7 @@
   "description": "Cross-platform utility library",
   "homepage": "https://github.com/absurdworlds/awlib",
   "license": "LGPL-3.0-or-later",
-  "supports": "!android",
+  "supports": "!uwp & !android",
   "dependencies": [
     {
       "name": "vcpkg-cmake",

--- a/ports/awlib/vcpkg.json
+++ b/ports/awlib/vcpkg.json
@@ -20,7 +20,8 @@
       "description": "Build graphics library",
       "supports": "!uwp",
       "dependencies": [
-        "glfw3"
+        "glfw3",
+        "libpng"
       ]
     },
     "hudf": {

--- a/ports/awlib/vcpkg.json
+++ b/ports/awlib/vcpkg.json
@@ -4,6 +4,7 @@
   "description": "Cross-platform utility library",
   "homepage": "https://github.com/absurdworlds/awlib",
   "license": "LGPL-3.0-or-later",
+  "supports": "!android",
   "dependencies": [
     {
       "name": "vcpkg-cmake",

